templates/components/llamaindex/typescript/documents/llamacloud/pipeline.ts

```diff
@@ -0,0 +1,16 @@
+import { Document } from "llamaindex";
+import { getDataSource } from "../../engine";
+
+export async function runPipeline(documents: Document[], filename: string) {
+  const currentIndex = await getDataSource();
+  for (const document of documents) {
+    document.metadata = {
+      ...document.metadata,
+      file_name: filename,
+      private: true, // mark document as private
+      use_local_url: true, // mark use local url to display the document
```

Comment:
@thucpn There's no code for this, so why did we add it?
if we will use this one, i'd like to call it as `use_remote_file` - the python is also using it to construct the file url correctly.